dandischema/datacite/__init__.py

```diff
@@ -78,16 +78,19 @@ def to_datacite(
     if publish:
         attributes["event"] = "publish"
 
-    attributes["identifiers"] = [
+    attributes["alternateIdentifiers"] = [
         # TODO: the first element is ignored, not sure how to fix it...
-        {"identifier": f"https://doi.org/{meta.doi}", "identifierType": "DOI"},
         {
-            "identifier": f"https://identifiers.org/{meta.id}",
-            "identifierType": "URL",
+            "alternateIdentifier": f"https://doi.org/{meta.doi}",
```

Comment:
Is this a DataCite DOI that you're minting? If so, it should go in the `doi` field. But it kind of depends on what your workflow is.

Reply:
;-)
Makes sense!  Will remove now!